testing/php7-zmq/APKBUILD

```diff
@@ -1,37 +1,38 @@
 # Contributor: Fabio Ribeiro <fabiorphp@gmail.com>
 # Maintainer: Fabio Ribeiro <fabiorphp@gmail.com>
 pkgname=php7-zmq
-_pkgreal=zmq
 pkgver=1.1.3
-_pkgver=${pkgver}
-pkgrel=0
-pkgdesc="ZeroMQ is a software library that lets you quickly design and implement a fast message-based applications."
+pkgrel=1
+_pkgver=${pkgver/_rc/RC}
+_phpver=${pkgname%%-*}
+_pkgreal=${pkgname#*-}
+pkgdesc="$_phpver extension: ZeroMQ messaging"
 url="http://pecl.php.net/package/$_pkgreal"
 arch="all"
 license="PHP"
-depends=
-pecldepends="php7-dev autoconf zeromq-dev"
-makedepends="$pecldepends"
-install=""
-subpackages=""
+pecldepends="$_phpver-dev autoconf"
+makedepends="$pecldepends zeromq-dev"
 source="http://pecl.php.net/get/$_pkgreal-$_pkgver.tgz"
-
-builddir="$srcdir"/$_pkgreal-$_pkgver
+builddir="$srcdir/$_pkgreal-$_pkgver"
 
 build() {
 	cd "$builddir"
-	phpize7 || return 1
-	./configure --prefix=/usr --with-php-config=php-config7 || return 1
-	make || return 1
+	phpize || return 1
+	./configure --prefix=/usr || return 1
+	make
+}
+
+check() {
+	make -C "$builddir" NO_INTERACTION=1 test
 }
 
 package() {
-	cd "$builddir"
-	make INSTALL_ROOT="$pkgdir"/ install || return 1
-	install -d "$pkgdir"/etc/php7/conf.d || return 1
-	echo "extension=$_pkgreal.so" > "$pkgdir"/etc/php7/conf.d/50_$_pkgreal.ini
+	local confdir=$(php-config|grep -o "scan-dir=[[:graph:]]*"|cut -d= -f2)
+	local apiver=$(($(phpize -v|awk -F: '/PHP Api/{print $2}')))
```

Comment:
What’s this? Why you use arithmetic expression with a command substitution inside? o.O